jib-core/src/main/java/com/google/cloud/tools/jib/configuration/BuildConfiguration.java

```diff
@@ -51,6 +53,7 @@
     private ImmutableList<LayerConfiguration> layerConfigurations = ImmutableList.of();
     private Class<? extends BuildableManifestTemplate> targetFormat = DEFAULT_TARGET_FORMAT;
     private String toolName = DEFAULT_TOOL_NAME;
+    @Nullable private EventEmitter eventEmitter;
```

Comment:
Could we have a `NonEmitter` that just silently drops all events?  Then we could avoid the `orElse` and null tests elsewhere.

Reply:
Yea, will do that.